docassemble/MOHUDEvictionProject/data/questions/defense_logic.yml

```
@@ -674,8 +696,8 @@ fields:
     choices:
       - A lease for a fixed period of time (like 1 year): fixed_period
       - A month-to-month, week-to-week or other lease that automatically renews: periodic
-      - I don't have have a written lease: no_written_lease
-      - I don't know: dont_know
+      - ${ "I don't have have a written lease" if person_answering == "tenant" else "The tenant does not have a written lease"}: no_written_lease
```

```suggestion
      - label: |
          % if if person_answering == "tenant":
          I don't have have a written lease
          % else:
          The tenant does not have a written lease
          % endif
          value: no_written_lease
```

Or use the approach in UpToCode, where we have a lookup dictionary for options that we want to display in the review block later as well.

Same for the below option.